pkg/serviceaccounts/controllers/docker_registry_service.go

```diff
@@ -218,7 +225,7 @@ func (e *DockerRegistryServiceController) watchForDockerURLChanges() {
 
 // getDockerRegistryLocations returns the dns form and the ip form of the secret
 func (e *DockerRegistryServiceController) getDockerRegistryLocations() []string {
-	ret := []string{}
+	ret := append([]string{}, e.additionalRegistryURLs...)
```

Comment:
why not start with ret := e.additionalRegistryURLs ?

Reply:
> why not start with ret := e.additionalRegistryURLs ?

fear of accidental mutation.